### PR TITLE
Added Cross-platform functionality for linux

### PR DESCRIPTION
**Pull request by adhyay2000**
Added the functionality for the Linux system. Also removed the error pointed in issue #1.

### CODE REVIEW

**Review by ieshaan12 — CHANGES_REQUESTED** [review]
Don't make a separate class for Linux Notification, make the if else for `platform.system()` within Notification.__init__()

File: TriggerClass.py [hunk 1]
```diff
@@ -74,7 +74,10 @@ def run(self):
                     else:
                         price = float(information['ask'])
                     if i.relation(price, float(i.value)):
-                        Notification(i, ticker['longName'])
+                        if platform.system() == 'Windows':
```
Comment (ieshaan12) [inline TriggerClass.py]: Remove the if...else from here

File: Notification.py [hunk 2]
```diff
@@ -1,7 +1,14 @@
 from win10toast import ToastNotifier
+import os
 
 
 class Notification:
     def __init__(self, trigger=None, CompanyName: str = ""):
         self.toast = ToastNotifier()
         self.toast.show_toast(trigger.name, f"{CompanyName}'s has been triggered, Price {trigger.relationName} {trigger.value}")
+
+class Notification_linux:
```
Comment (ieshaan12) [inline Notification.py]: Separate class not needed

File: Notification.py [hunk 3]
```diff
@@ -1,7 +1,14 @@
 from win10toast import ToastNotifier
+import os
 
 
 class Notification:
     def __init__(self, trigger=None, CompanyName: str = ""):
         self.toast = ToastNotifier()
```
Comment (ieshaan12) [inline Notification.py]: Add if else for Windows and Linux here

File: Notification.py [hunk 4]
```diff
@@ -1,7 +1,14 @@
 from win10toast import ToastNotifier
```
Comment (ieshaan12) [inline Notification.py]: if...else for Windows and Linux while importing, no unnecessary imports.

**Review by ieshaan12 — CHANGES_REQUESTED** [review]
Make these changes!

**Comment by adhyay2000** [comment]
will do the required changes.

**Comment by adhyay2000** [comment]
Hey, I have modified the code according to the changes requested. Please have a look and let me know if there are any additional changes.

**Pull request merged**